.github/workflows/ci.yml

```diff
@@ -340,10 +340,12 @@ jobs:
 
       - name: Install test deps
         run: |
-          & build\bin\nvim.exe "--version"
+          $PSNativeCommandArgumentPassing = "Standard"
+
+          .\build\bin\nvim.exe --version
 
           # Ensure that the "win32" feature is set.
-          & build\bin\nvim -u NONE --headless -c 'exe !has(\"win32\").\"cq\"'
```

Comment:
The call operator seems no longer needed as there are no variables in this command, and it needs to be removed for the `$PSNativeCommandArgumentPassing = "Standard"` to work.